src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.13.0-preview</Version>
+		<Version>2.0.0-preview-01</Version>
```

Comment:
@shahabhijeet  Is this a recommended versioning pattern

Reply:
Discussed by email - Scott said to leave it at 1.13.0